core/src/components/radio/radio.ionic.scss

```diff
@@ -0,0 +1,182 @@
+  --focus-ring-width: #{globals.$ionic-border-size-050};
+
+  @include globals.typography(globals.$ionic-body-md-regular);
+  min-height: globals.$ionic-scale-1200;
+
+  color: globals.$ionic-color-neutral-1200;
+}
+
+// Radio Color
+// --------------------------------------------------
+
+:host(.ion-color.radio-checked) .radio-icon {
+  border-color: globals.current-color(base);
+
+  background-color: globals.current-color(base);
+}
+
+// Radio Label
+// ----------------------------------------------------------------
+
+:host(.in-item) .label-text-wrapper {
+  @include globals.margin(globals.$ionic-space-250, null, globals.$ionic-space-250, null);
+}
+
+:host(.in-item.radio-label-placement-stacked) .label-text-wrapper {
+  @include globals.margin(globals.$ionic-space-250, null, globals.$ionic-space-400, null);
+}
+
+:host(.in-item.radio-label-placement-stacked) .native-wrapper {
+  @include globals.margin(null, null, globals.$ionic-space-250, null);
+}
+
+// Radio Native Wrapper
+// ----------------------------------------------------------------
+
+.native-wrapper .radio-icon {
+  width: globals.$ionic-scale-600;
+  height: globals.$ionic-scale-600;
+}
+
+// Ionic Radio Outer Circle: Unchecked
+// -----------------------------------------
+
+.radio-icon {
+  @include globals.margin(0);
+  @include globals.border-radius(var(--border-radius));
+
+  border-width: var(--border-width);
+  border-style: var(--border-style);
+  border-color: var(--color);
+
+  background-color: globals.$ionic-color-base-white;
+}
+
+// Ionic Radio Inner Circle: Unchecked
+// -----------------------------------------
+
+.radio-inner {
+  @include globals.border-radius(var(--inner-border-radius));
+
+  width: calc(32% + var(--border-width));
+  height: calc(32% + var(--border-width));
+
+  background-color: globals.$ionic-color-base-white;
+}
+
+// Ionic Radio Outer Circle: Checked
+// -----------------------------------------
+
+:host(.radio-checked) .radio-icon {
+  border-color: var(--color-checked);
+
+  background-color: var(--color-checked);
+}
+
+// Radio Label Placement - Start
+// ----------------------------------------------------------------
+
+:host(.radio-label-placement-start) .label-text-wrapper {
+  /**
+   * The margin between the label and
+   * the radio should be on the end
+   * when the label sits at the start.
+   */
+  @include globals.margin(null, globals.$ionic-space-400, null, 0);
+}
+
+// Radio Label Placement - End
+// ----------------------------------------------------------------
+
+/**
+ * The margin between the label and
+ * the radio should be on the start
+ * when the label sits at the end.
+ */
+:host(.radio-label-placement-end) .label-text-wrapper {
+  @include globals.margin(null, 0, null, globals.$ionic-space-400);
+}
+
+// Radio Label Placement - Fixed
+// ----------------------------------------------------------------
+
+:host(.radio-label-placement-fixed) .label-text-wrapper {
```

Comment:
Should this be combined with `:host(.radio-label-placement-start) .label-text-wrapper {`?

Reply:
https://github.com/ionic-team/ionic-framework/pull/29805/commits/59ea4906cac4aefa26d87a8f5e3a6a5d830ee160